docs_zh-CN/conf.py

```diff
@@ -33,8 +33,8 @@
 ]
 
 autodoc_mock_imports = [
```

Comment:
No its the mock imports, but the extensions. Please double check the referenced pr

Reply:
done.